source/openapi-admin-v3.yaml

```diff
@@ -6454,6 +6494,21 @@ components:
         hash:
           type: string
           description: The MD5 checksum hash for the hosted asset
+    RefreshTokenExpiration:
+      type: object
+      properties:
+        expiration_time_seconds:
+          type: number
+          minimum: 1800
+          maximum: 15552000
+          default: 5184000
+          description: |-
+            The time in seconds that a user session refresh
+            token is valid for once issued. After this time, the
```

Comment:
```suggestion
            The time in seconds that a user session refresh
            token is valid for since it has been issued. After this time, the
```